public/app/features/plugins/admin/components/PluginListBadges.tsx

```diff
@@ -38,6 +38,7 @@ function EnterpriseBadge({ id }: { id: string }) {
 
   return (
     <HorizontalGroup>
+      <PluginSignatureBadge status={plugin.signature} />
```

Comment:
Is it possible to have unsigned Enterprise plugins? I'd decided previously not to include the `PluginSignatureBadge` here as I'd assumed the lock icon on the Enterprise badge was acting like a "signature" type.

Reply:
I see mate. Yea, I think unfortunately it is possible.
IIRC the backend part of a enterprise plugin won't launch if the plugin is not signed, but the frontend part will.

Although maybe they have changed this in Grafana Enterprise, will double check 👍